src/components/DataDimension/DropZone.js

```diff
@@ -0,0 +1,39 @@
+import { useDroppable } from '@dnd-kit/core'
+import cx from 'classnames'
+import PropTypes from 'prop-types'
+import React from 'react'
+import { FIRST_DROPZONE_ID } from './constants.js'
+import styles from './styles/DropZone.style.js'
+
+const DropZone = ({ firstElementId, overLastDropZone }) => {
+    const { isOver, setNodeRef, active } = useDroppable({
+        id: FIRST_DROPZONE_ID,
+    })
+
+    let draggingOver = false
+    if (overLastDropZone && !firstElementId) {
+        draggingOver = true
+    } else {
+        draggingOver = firstElementId === active?.id ? false : isOver
+    }
+
+    return (
+        <>
+            <div
+                ref={setNodeRef}
+                className={cx('first-dropzone', {
+                    over: draggingOver,
```

Comment:
As I understand it `over` is dnd kit lingo, but for our class name maybe we can use something more descriptive? (and less dnd-kit'y)

Reply:
'dragging-over'